project/Dependencies.scala

```diff
@@ -61,12 +60,14 @@ object Dependencies {
 
   val akkaHttp = Seq(
     "com.typesafe.akka" %% "akka-http"            % AkkaHttp,
+    "com.typesafe.akka" %% "akka-http-caching"    % AkkaHttp,
```

Comment:
I'm pretty sure the list of modules [here](https://developer.lightbend.com/docs/introduction/getting-help/build-dependencies.html#_akka_http) is incomplete.

Reply:
That list is incomplete (and not just for Akka HTTP). Just noticed there are other Lagom artifacts missing too.

Reply:
I've raised https://github.com/lightbend/lightbend-technology-intro-doc/issues/240